### PR TITLE
Version 6.0.1 release blog post

### PR DESCRIPTION
**Pull request by amol-**

### CODE REVIEW

File: _posts/2021-11-22-6.0.1-release.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+* Dataset api now supports `existing_data_behavior` option when writing datasets.
+* Installing pyarrow from source distribution now works with setuptools 58.5
+
+## R notes
+
+* Fixed a crash when summarizing after filtering to no rows
+* Added bindings for `str_count`
+
+For more details, see the [complete R changelog][4]. 
+
```
Comment (zeroshade) [inline _posts/2021-11-22-6.0.1-release.md:65]:
```suggestion
## Go notes

* Arrow and Parquet modules are properly brought together under a top level `github.com/apache/arrow/go` module which can be installed as `github.com/apache/arrow/go/v6/arrow@v6.0.1`.
```

**Review by pitrou — APPROVED** [review]
+1

File: _posts/2021-11-22-6.0.1-release.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+## Go notes
+
+* Arrow and Parquet modules are properly brought together under a top level `github.com/apache/arrow/go` module which can be installed as `github.com/apache/arrow/go/v6/arrow@v6.0.1`.
+
```
Comment (bkmgit) [inline _posts/2021-11-22-6.0.1-release.md:69]:
```suggestion
## Ruby notes

Thanks to [simpl1g](https://github.com/simpl1g) Ruby now supports explicit initialization of S3 APIs. An introduction
to using Red Arrow is available in the [main repository](https://github.com/apache/arrow/blob/master/ruby/README.md), look for more updates in upcoming releases.
```

Comment (kou) [reply _posts/2021-11-22-6.0.1-release.md:69]: 6.0.1 doesn't include the feature.
7.0.0 will ship it.

Comment (bkmgit) [reply _posts/2021-11-22-6.0.1-release.md:69]: Ok. Maybe still helpful to highlight the introduction to using Ruby? This is not included in the release, but it is very helpful for anyone looking to get started with the Ruby bindings.

Comment (kou) [reply _posts/2021-11-22-6.0.1-release.md:69]: 6.0.1 is a patch release.
We should not do it for a patch release. Release annonces for patch releases should focus on only bug fixes.

Comment (bkmgit) [reply _posts/2021-11-22-6.0.1-release.md:69]: Ok, thanks.

File: _posts/2021-11-22-6.0.1-release.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+## Ruby notes
+
+Thanks to [simpl1g](https://github.com/simpl1g) Ruby now supports explicit initialization of S3 APIs. 
+An introduction to using Red Arrow is available in the [main repository](https://github.com/apache/arrow/blob/master/ruby/README.md), 
+look for more updates in upcoming releases.
```
Comment (bkmgit) [inline _posts/2021-11-22-6.0.1-release.md:74]: No updates for Ruby in this release, so can remove content.
```suggestion
```

**Comment by nealrichardson** [comment]
@domoritz do you want to add a note about the JS fixes? I think that was one of the reasons for doing the patch release.

**Pull request merged**